src/mqt/predictor/ml/helper.py

```diff
@@ -11,7 +11,16 @@
 from joblib import dump
 from qiskit import QuantumCircuit
 
-from mqt.predictor import utils
+from mqt.predictor import ml, utils
+
+
+def compile_prediction(qc: QuantumCircuit):
```

Comment:
this function could use a return type annotation.
We should really include `mypy` in the pre-commit checks so that full typing annotations are maintained throughout the library.

Reply:
Added the return type annotation (and mypy integration as an issue).